src/vendors/VendorsScreen.tsx

```diff
@@ -33,6 +45,18 @@ export default function VendorsScreen() {
     setCurrentVendor(null)
   }
 
+  const onRefresh = async () => {
+    dispatch(fetchVendors())
+  }
+
+  const refresh: React.ReactElement<RefreshControlProps> = (
+    <RefreshControl refreshing={isLoading} onRefresh={onRefresh} colors={[colors.greenUI]} />
```

Comment:
This refreshable component should span the entire viewport height.

Reply:
Gotcha, i copied this from homepage so you should also fix that on the homepage then.

Reply:
Im checking now and it works identical to the homepage. Are you sure you don't want it like this?